packages/fuel-indexer/src/service.rs

```diff
@@ -63,7 +63,10 @@ impl IndexerService {
         )
         .await?;
 
+        println!("Registered Index({})", &manifest.uid());
```

Comment:
print statements are still here.  If these are informative let's change them to `info!`. If they were just used for testing, let's remove them?

Reply:
My bad, I removed them. But they turned up again after I copied in my older commit.